image-layout.md

```diff
@@ -156,4 +127,79 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## oci-layout file
+
+This JSON object serves as a marker for the base of an Open Container Image Layout and to provide the version of the image-layout in use.
+The `imageLayoutVersion` value will align with the OCI Image Specification version at the time changes to the layout are made, and will pin a given version until changes to the image layout are required.
+
+### oci-layout Example
+
+```json
+{
+    "imageLayoutVersion": "1.0.0"
+}
+```
+
+## index.json file
+
+This REQUIRED file is the entry point for references and descriptors of the image-layout.
+The [manifest-list](manifest-list.md) is an multi-descriptor entry point.
+It is intended to be generated from the content in an image-layout after any changes.
```

Comment:
This line makes less sense to me now that we're dropping `refs/`.  Can we drop it?

And one line up, “an multi-descriptor” → “a multi-descriptor”.